src/pybind/mgr/dashboard/services/access_control.py

```diff
@@ -283,6 +293,13 @@ def from_dict(cls, r_dict):
     GANESHA_MGR_ROLE.name: GANESHA_MGR_ROLE,
 }
 
+# static name-like roles list for role mapping
+SYSTEM_ROLES_NAMES = {
+    ADMIN_ROLE: [ADMIN_ROLE.name, 'admin'],
+    READ_ONLY_ROLE: [READ_ONLY_ROLE.name, 'read', 'guest', 'monitor']
+}
```

Comment:
This is ok. If you want to make it more flexible, you can check [how Grafana does OAuth2 role mapping](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/generic-oauth/#configure-role-mapping). We have support in CentOS 9 for a [Python JMESPath parser](https://pypi.org/project/jmespath/).

Reply:
For now I have removed this, we'll look into itroducing something similar like Grafana in the future